public/app/core/utils/fetch.ts

```diff
@@ -89,6 +90,9 @@ export const parseBody = (options: BackendSrvRequest, isAppJson: boolean) => {
   if (!options.data || typeof options.data === 'string') {
     return options.data;
   }
+  if (options.data.constructor.name === 'Blob') {
```

Comment:
does `options.data instanceof Blob` work here?

Reply:
No apparently jest need a polyfill but I'm not sure how to do it.

Reply:
@joshhunt should we add a polyfill for this? see https://github.com/jsdom/jsdom/issues/2555 and https://github.com/elastic/kibana/pull/162197

Reply:
Ok seems like there actually is a Blob polyfill https://github.com/grafana/grafana/blob/e7797ac4395b431715810315c3137b9e1913f509/public/test/jest-setup.ts#L9.

Reply:
I wasn't sure how to use it properly.